assets/js/googlesitekit-data.js

```diff
@@ -1,10 +1,22 @@
 /**
  * External dependencies
  */
-import Data from 'assets/js/googlesitekit/data';
+
+/**
+ * Internal dependencies
+ */
+import SiteKitRegistry from 'assets/js/googlesitekit/data';
+import { collect, collectReducers } from 'assets/js/googlesitekit/data/utils';
 
 if ( typeof global.googlesitekit === 'undefined' ) {
 	global.googlesitekit = {};
 }
 
-global.googlesitekit.data = Data;
+const registry = SiteKitRegistry;
+if ( typeof global.googlesitekit.data === 'undefined' ) {
+	global.googlesitekit.data = { collect, collectReducers, registry };
```

Comment:
We shouldn't have a `registry` property on data, one should be able to call the functions directly on `googlesitekit.data`. I suggest to include utility functions like that directly on the registry, or alternatively in a completely separate package.

Furthermore, let's create the object that gets assigned to `global.googlesitekit.data` above this clause, and then export exactly that object further down.